client/implementation-vertx/pom.xml

```diff
@@ -71,4 +71,31 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>default-test</id>
+                        <configuration>
+                            <excludes>TypesafeTckClientModelSuite</excludes>
+                        </configuration>
+                    </execution>
+                    <execution>
```

Comment:
Please add a comment explaining why we're doing this, otherwise I won't remember it after two weeks 
(that junit doesn't seem to like executing the same tests separately in multiple suites or something)